### ISSUE TITLE
Wishlist: simple integration with for mass creation of domains or migration

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Spacefish/powerdns-webinterface#8]
```
It would be nice to have either functions ready to be integrated into existing 
platforms or to be used with simple wrappers to create command line management 
with the same functionality and according to the same rules that exist in the 
web interface.

examples; add domain based on template X ; reset user password to Y; add record 
a ttt.xxx 1.1.1.1 ...etc ; delete domain; delete record ...etc.

it would be useful only if it does not depend on numeric IDs of the entries in 
the database; thus the functions can be sets and gets where one would be able 
to build scripts for automation around it.
```

Original issue reported on code.google.com by `maysara....@gmail.com` on 30 Sep 2011 at 4:06
- Blocked on: #9
